renderers/main.py

```diff
@@ -29,104 +33,59 @@ def __init__(self, matrix, data):
-            if self.data.standings.is_postseason():
-                standings.render_bracket(
-                    self.canvas,
-                    self.data.config.layout,
-                    self.data.config.scoreboard_colors,
-                    self.data.standings.leagues[self.standings_league],
-                )
-            else:
-                standings.render_standings(
-                    self.canvas,
-                    self.data.config.layout,
-                    self.data.config.scoreboard_colors,
-                    self.data.standings.current_standings(),
-                    self.standings_stat,
-                )
-
-            if self.data.network_issues:
-                network.render_network_error(self.canvas, self.data.config.layout, self.data.config.scoreboard_colors)
-
-            self.canvas = self.matrix.SwapOnVSync(self.canvas)
+        self.__draw_standings(permanent_cond)
 
-            if self.data.standings.is_postseason():
-                time.sleep(20)
-                if self.standings_league == "NL":
-                    self.standings_league = "AL"
-                else:
-                    self.standings_league = "NL"
-            else:
-                # we actually keep this logic on the render thread
-                if self.canvas.width > 32:
-                    time.sleep(10)
-                    self.data.standings.advance_to_next_standings()
-                else:
-                    if self.standings_stat == "w":
-                        self.standings_stat = "l"
-                    else:
-                        self.standings_stat = "w"
-                        self.data.standings.advance_to_next_standings()
-                    time.sleep(5)
+        # Out of season off days don't always return standings so fall back on the news renderer
+        debug.error("No standings data.  Falling back to news.")
+        self.__draw_news(permanent_cond)
 
     # Renders a game screen based on it's status
-    def __render_game(self) -> NoReturn:
-        # Set the refresh rate
+    # May also call draw_offday or draw_standings if there are no games
+    def __render_gameday(self) -> NoReturn:
         refresh_rate = self.data.config.scrolling_speed
-
         while True:
-            self.__draw_standings()
+            if not self.data.schedule.games_live():
+                if self.data.config.news_no_games and self.data.config.standings_no_games:
+                    self.__draw_news(
+                        all_of(timer_cond(STANDINGS_NEWS_SWITCH_TIME, refresh=refresh_rate), self.no_games_cond)
+                    )
+                    self.__draw_standings(all_of(timer_cond(STANDINGS_NEWS_SWITCH_TIME), self.no_games_cond))
+                    continue
+                elif self.data.config.news_no_games:
+                    self.__draw_news(self.no_games_cond)
+                elif self.data.config.standings_no_games:
+                    self.__draw_standings(self.no_games_cond)
```

Comment:
Do we need to `continue` out of these 2 conditions also?

Reply:
Not necessarily- when those functions return, it's because a game is live, so we do want to go directly into drawing it.